### PR TITLE
RFC-0301 - Technical Editing

### PR DESCRIPTION
**Pull request by anselld**
## Description
Technical editing.

## Types of changes

* [ ] Bug fix (non-breaking change which fixes an issue)
* [ ] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Feature refactor (No new feature or functional changes, but performance or technical debt improvements)
* [ ] New Tests
* [x] Documentation

## Checklist:

* [x] I'm merging against the `development` branch
* [ ] I ran `cargo-fmt --all` before pushing
* [ ] My change requires a change to the documentation.
* [ ] I have updated the documentation accordingly.
* [ ] I have added tests to cover my changes.

### CODE REVIEW

**Comment by anselld** [comment]
Hi @hansieodendaal , if you get a chance, please help. Something is wrong with this PR. It is a new PR and the first two commits in the above list were on other documents.
Regarding the third commit, I have the following comment: in the code, the word 'characters' appears as 'charaters'. Is that okay?

File: RFC/src/RFC-0301_NamespaceRegistration.md [hunk 1]
```diff
@@ -142,14 +146,15 @@ linked to a [digital asset] on the DAN.
 | OpenAlias TXT DNS Record Field | OpenAlias TXT DNS Record Data                                |
 | ------------------------------ | ------------------------------------------------------------ |
 | oa1:\<name\>                   | "oa1:tari"                                                   |
-| pk                             | \<256 bit public key, in hexadecimal format (64 characters), that is converted into a `Base58` encoded string (44 characters)\> |
-| raid_id                        | \<`RAID_ID` (*see [The RAID_ID](#the-raid_id)*) (15 characters)\> |
-| nonce                          | \<256 bit public nonce, in hexadecimal format (64 characters), that is converted into a `Base58` encoded string (44 characters)\> |
-| sig                            | \<[Asset issuer]'s 256 bit Schnorr signature for the `RAID_ID` (*see [The RAID_ID](#the-raid_id)*), in hexadecimal format (64 characters), that is converted into a `Base58` encoded string (44 characters)\> |
+| pk                             | \<256 bit public key, in hexadecimal format (64 characters), that is converted into a `Base58` encoded string (44 characters)\>. |
```
Comment (hansieodendaal) [inline RFC/src/RFC-0301_NamespaceRegistration.md]:
```suggestion
| pk                             | \<256 bit public key, in hexadecimal format (64 characters), that is converted into a `Base58` encoded string (44 characters)\> |
```

File: RFC/src/RFC-0301_NamespaceRegistration.md [hunk 2]
```diff
@@ -142,14 +146,15 @@ linked to a [digital asset] on the DAN.
...
+| raid_id                        | \<`RAID_ID` (*refer to [RAID_ID](#raid_id)*) (15 characters)\>. |
```
Comment (hansieodendaal) [inline RFC/src/RFC-0301_NamespaceRegistration.md]:
```suggestion
| raid_id                        | \<`RAID_ID` (*refer to [RAID_ID](#raid_id)*) (15 characters)\> |
```

File: RFC/src/RFC-0301_NamespaceRegistration.md [hunk 3]
```diff
@@ -142,14 +146,15 @@ linked to a [digital asset] on the DAN.
...
+| nonce                          | \<256 bit public nonce, in hexadecimal format (64 characters), that is converted into a `Base58` encoded string (44 characters)\>. |
```
Comment (hansieodendaal) [inline RFC/src/RFC-0301_NamespaceRegistration.md]:
```suggestion
| nonce                          | \<256 bit public nonce, in hexadecimal format (64 characters), that is converted into a `Base58` encoded string (44 characters)\> |
```

File: RFC/src/RFC-0301_NamespaceRegistration.md [hunk 4]
```diff
@@ -142,14 +146,15 @@ linked to a [digital asset] on the DAN.
...
+| sig                            | \<[Asset issuer]'s 256 bit Schnorr signature for the `RAID_ID` (*refer to [RAID_ID](#raid_id)*), in hexadecimal format (64 characters), that is converted into a `Base58` encoded string (44 characters)\>. |
```
Comment (hansieodendaal) [inline RFC/src/RFC-0301_NamespaceRegistration.md]:
```suggestion
| sig                            | \<[Asset issuer]'s 256 bit Schnorr signature for the `RAID_ID` (*refer to [RAID_ID](#raid_id)*), in hexadecimal format (64 characters), that is converted into a `Base58` encoded string (44 characters)\> |
```

File: RFC/src/RFC-0301_NamespaceRegistration.md [hunk 5]
```diff
@@ -142,14 +146,15 @@ linked to a [digital asset] on the DAN.
...
 | desc                           | \<Optional RAID description\>; ASCII String; Up to 48 characters for the condensed version (using only one string) and up to 235 characters (when spanning two strings). |
-| crc                            | \<CRC-32 checksum of the entire record up to but excluding the checksum key-value pair (starting at "**oa1:tari**" and ending at the last "**;**" before the checksum key-value pair) in hexadecimal format (8 characters)\> |
+| crc                            | \<CRC-32 checksum of the entire record, up to but excluding the checksum key-value pair (starting at "**oa1:tari**" and ending at the last "**;**" before the checksum key-value pair) in hexadecimal format (eight characters)\>. |
```
Comment (hansieodendaal) [inline RFC/src/RFC-0301_NamespaceRegistration.md]:
```suggestion
| crc                            | \<CRC-32 checksum of the entire record, up to but excluding the checksum key-value pair (starting at "**oa1:tari**" and ending at the last "**;**" before the checksum key-value pair) in hexadecimal format (eight characters)\> |
```

File: RFC/src/RFC-0301_NamespaceRegistration.md [hunk 6]
```diff
@@ -142,14 +146,15 @@ linked to a [digital asset] on the DAN.
...
 
-&nbsp;&nbsp;&nbsp;&nbsp;Examples: Two example OpenAlias TXT DNS records are shown; the first a condensed version and the second one that spans two strings:
+&nbsp;&nbsp;&nbsp;&nbsp;**Examples:** Two example OpenAlias TXT DNS records are shown; the first is a condensed version and 
+the second spans two strings:
```
Comment (hansieodendaal) [inline RFC/src/RFC-0301_NamespaceRegistration.md]: Please replace the 1st string with:

```
OpenAlias TXT DNS record (condensed: 212 characters):
IN   TXT = "oa1:tari pk=EcbmnM6PLosBzpyCcBz1TikpNXRKcucpm73ez6xYfLtg;id=RYqMMuSmBZFQkgp;
nonce=5ctFNnCfBrP99rT1AFmj1WPyMD8uAdNUTESHhLoV3KBZ;sig=9TxTorviyTJAaVJ4eY4AQPixwLb6SDL4dieHff6MFUha;
desc=Cartoon characters;crc=176BE80C"
```

Comment (hansieodendaal) [reply RFC/src/RFC-0301_NamespaceRegistration.md]: Please replace the 2nd string with:

```
OpenAlias TXT DNS record (spanning two strings: string 1 = 179 characters and string 2 = 250 characters):
IN   TXT = "oa1:tari pk=EcbmnM6PLosBzpyCcBz1TikpNXRKcucpm73ez6xYfLtg; id=RYqMMuSmBZFQkgp; 
nonce=5ctFNnCfBrP99rT1AFmj1WPyMD8uAdNUTESHhLoV3KBZ; sig=9TxTorviyTJAaVJ4eY4AQPixwLb6SDL4dieHff6MFUha; "
"desc=Cartoon characters: Mickey Mouse\; Minnie Mouse\; Goofy\; Donald Duck\; Pluto\; Daisy Duck\; 
Scrooge McDuck\; Launchpad McQuack\; Huey, Dewey and Louie\; Bambi\; Thumper\; Flower\; Faline\; 
Tinker Bell\; Peter Pan and Captain Hook.; crc=54465902"
```

Comment (anselld) [reply RFC/src/RFC-0301_NamespaceRegistration.md]: Hi @hansieodendaal, thanks - will do. I see no 'button' here, so I will copy the strings into the md files.

Comment (anselld) [reply RFC/src/RFC-0301_NamespaceRegistration.md]: Hi @hansieodendaal , I've accepted your changes, have copied the strings into the md file and have pushed the md file.

Comment (anselld) [reply RFC/src/RFC-0301_NamespaceRegistration.md]: Hi @hansieodendaal , I've also combined the commits.

**Pull request merged**